test/uk/gov/hmrc/cbcr/services/DataMigrationCriteriaSpec.scala

```diff
@@ -27,7 +27,7 @@ class DataMigrationCriteriaSpec extends UnitSpec with GuiceOneAppPerSuite {
     import DataMigrationCriteria._
 
     "produce an jsObject when result cannot get the correct values from the configuration" in {
-      val failCaseJson = Json.obj("cbcId" -> Json.obj("$regex" -> "X[A-Z]CBC00.*"))
+      val failCaseJson = Json.obj("cbcId" -> Json.obj(f"$$regex" -> "X[A-Z]CBC00.*"))
```

Comment:
Why this f interpolator?

Reply:
It was giving an error for missing interpolator. Found on internet that this approach is fixing the warning.

Reply:
Sorry, a warning not error.

Reply:
https://stackoverflow.com/questions/39401213/disable-false-warning-possible-missing-interpolator/39401810

Reply:
I see

To be honest, the first thing I had to do was look up what an f interpolator did!

I'd be uncomfortable with a workaround like that because it gives no idea as to why it is like that & you make yourself a hostage to fortune for someone implementing a check of say interpolated string when no interpolation done.

The real problem is that you want to get Matchers.regex out of scope there, it is a name from somewhere else entirely that you have no need of.

As this is rare, I would be in inclined to at least put a comment with a link to the StackExchange, so that anyone else can see why.

Reply:
Will add the link in a comment. As you say this will be very rare.